tools/internal/ios/scripts/processAppCenterLogs.js

```diff
@@ -17,8 +17,8 @@ const api_token = process.argv[4];  // app_center_token;
 const test_id = require("../run_output.json")[0].testRunId; // TODO: Pass test results path
 
 const xmlFilter = "[Mocha_Result_XML]: ";
-const deviceLogsPath = "ios/device_logs.txt"
-const resultsPath = "ios/junit_results.xml"
+const deviceLogsPath = "device_logs.txt"
```

Comment:
where do these files end up?

Reply:
@calebmshafer in the "ios" directory. Would it be better to keep them out of the source files and instead store in lib?

Reply:
I think storing it in lib would be fine, just wanted to make sure it went under each package's lib folder and not in some other one.

Reply:
I added a template parameter and updated the scripts to make these paths clearer.